### DISCUSSION TITLE
Not one to complain often about something, but why is broodmother so easily countered?

### DISCUSSION THREAD

**Discussion by kyhmr** [ValveSoftware/Dota2-Gameplay#14510] [General]
Am I playing broodmother wrong or what is going on?
The spiders are a complete pushover, one enemy battlefury or AOE spell clears them which also makes your bola web spit useless, and when you think your hunger ability is going to have an effect, you realise your hero has no armor and not the greatest damage.

On dotabuff she has a 42% winrate so is she just a terribly balanced hero or is it one of those PRO LEAGUE ONLY heroes that normal players are not allowed to enjoy?

**Comment by commentator24** [comment]
brood is one of the strongest hero at lv 6, great at hiding in trees and ambush enemies from behind.
bfury takes about 15-16 min to complete so you have min 6-15 to take advantage of it.
avoid heroes that can deal aoe, just pretend youre going to that lane then quickly switch lane with gate or tp whenever you see heroes that you cant kill quickly OR find lanes that only has 1 hero. usually I try to overwhelm the enemy safelane and pick off their supports first. the goal is to go aggressive when your creeps reach enemy towers and make enemy safelane so scared that they can't even get XP at tower and finally jungle. 

maybe you can describe how you play brood? usually i just max web spit, rush soulring and phylactery to overwhelm the enemies early game. then orchid bkb. or you can simply rush bloodthorn to deal huge dmg with creeps

**Comment by nyczalex** [comment]
I just think he's more of a team coordinated hero, that's all.. Pro leagues understand while the regular norms don't.. You can take their whole t1 and t2's with 1 hand but getting all the raxs is more challenging, more of a coin flip on who has the better team which is fine because dota is a TEAM game.

**Comment by mrstealyommr** [comment]
She farms very fast and is pretty much unkillable without stuns. Team without aoe spells automatically lose. By the time you would have bfury youd have lost all towers and map control. She remains a cheese pick.
